myst_nbs/gaussian_processes/gaussian_process.myst.md

```diff
@@ -6,48 +6,43 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3
+  display_name: Python 3.10.5 ('pymc-dev')
   language: python
   name: python3
 ---
 
+(gaussian_process)=
 # Gaussian Processes using numpy kernel
 
-(c) 2016 by Chris Fonnesbeck
+:::{post} 2016
+:tags: gaussian processes, 
+:category: advanced
+:author: Chris Fonnesbeck
+:::
 
 +++
 
-Example of simple GP fit, adapted from Stan's [example-models repository](https://github.com/stan-dev/example-models/blob/master/misc/gaussian-process/gp-fit.stan).
+Example of simple Gaussian Process fit, adapted from Stan's [example-models repository](https://github.com/stan-dev/example-models/blob/master/misc/gaussian-process/gp-fit.stan).
 
-This example builds a Gaussian process from scratch, to illustrate the underlying model. PyMC3 now includes a dedicated GP submodule which is going to be more usable for a wider variety of problems.
+This example builds a Gaussian process from scratch, to illustrate the underlying model. PyMC now includes a dedicated Gaussian Process submodule which is going to be more usable for a wider variety of problems.
 
 ```{code-cell} ipython3
+import aesara.tensor as at
 import arviz as az
 import matplotlib.pyplot as plt
 import numpy as np
-import pymc3 as pm
+import pymc as pm
 import seaborn as sns
-import theano.tensor as T
-
-from pymc3 import (
-    NUTS,
-    Deterministic,
-    HalfCauchy,
-    Model,
-    MvNormal,
-    find_MAP,
-    sample,
-    summary,
-    traceplot,
-)
-from theano import shared
-from theano.tensor.nlinalg import matrix_inverse
 
-print(f"Running on PyMC3 v{pm.__version__}")
+from aesara import shared
```

Comment:
can probably use `pm.MutableData`

Reply:
I realized this was not used in the notebook, so I deleted.